src/OAuth2.php

```diff
@@ -703,6 +705,16 @@ public function getCacheKey()
         return null;
     }
 
+    /**
+     * Gets this instance's SubjectTokenFetcher
+     *
+     * @return null|ExternalAccountCredentialSourceInterface
+     */
+    public function getSubjectTokenFetcher(): null|ExternalAccountCredentialSourceInterface
```

Comment:
It's more standard to write this like this:
```suggestion
    public function getSubjectTokenFetcher(): ?ExternalAccountCredentialSourceInterface
```